HLTrigger/HLTcore/plugins/HLTPrescaleExample.cc

```diff
@@ -39,12 +39,12 @@ void HLTPrescaleExample::analyze(edm::Event const& iEvent, edm::EventSetup const
   auto hltPSFrac = hltPSProvider_.prescaleValue<FractionalPrescale>(iEvent, iSetup, hltPath_);
 
   auto l1HLTPSDouble = hltPSProvider_.prescaleValues<double>(iEvent, iSetup, hltPath_);
-  auto l1HLTPSInt = hltPSProvider_.prescaleValues<int>(iEvent, iSetup, hltPath_);
+  auto l1HLTPSInt = hltPSProvider_.prescaleValues<double, int>(iEvent, iSetup, hltPath_);
   auto l1HLTPSFrac = hltPSProvider_.prescaleValues<FractionalPrescale>(iEvent, iSetup, hltPath_);
   auto l1HLTPSDoubleFrac = hltPSProvider_.prescaleValues<double, FractionalPrescale>(iEvent, iSetup, hltPath_);
 
   auto l1HLTDetailPSDouble = hltPSProvider_.prescaleValuesInDetail<double>(iEvent, iSetup, hltPath_);
-  auto l1HLTDetailPSInt = hltPSProvider_.prescaleValuesInDetail<int>(iEvent, iSetup, hltPath_);
+  auto l1HLTDetailPSInt = hltPSProvider_.prescaleValuesInDetail<double, int>(iEvent, iSetup, hltPath_);
   auto l1HLTDetailPSFrac = hltPSProvider_.prescaleValuesInDetail<FractionalPrescale>(iEvent, iSetup, hltPath_);
 
   std::cout << "---------Begin Event--------" << std::endl;
```

Comment:
so I disagree with this comment and I will explain my reasoning. 

The HLTPrescaleExample module is clearly a test/example module which will never ever run in normal CMSSW jobs as it doesnt do anything.

Its purpose is to help users hence the simplicity. 

Even though I have 15 years on CMSSW, I still find configuring the message logger properly a bit of trial and error. Perhaps this is just me but perhaps not. But I figure my target audience, a new grad student has no hope. 

If this was to run other than test/example, I would fully agree with changing it but given its purpose is a simple example to try and quickly help users who dont want to waste time dealing with the message logger, I would have left it as is.

Reply:
I can revert this change. Regarding the cfg file, what would you prefer? (keep its update or not, use it or not as a unit test)

Reply:
ah yes you are now using it as a unit test which means it probably should be message logger.  Anyways thought I would explain why it was done with couts in the first place. 

The issue that it would be very nice that this example prints out some output so we would need to configure the example properly configure the message logger.

I envisioned the user running it as a test and then having the prescales dump to screen which I think is usually useful for learning and getting a feel if its working.

Reply:
I see. Indeed, I didn't touch it initially, then I updated it such that `cmsRun hltPrescaleExample_cfg.py` (which is all the unit test is) gives the full printout, _a la_ `cout`. I leave it as is, unless you let me know otherwise.